src/oaklib/utilities/lexical/lexical_indexer.py

```diff
@@ -124,7 +125,7 @@ def create_lexical_index(
                     term2 = term
                     for tr in pipeline.transformations:
                         if tr.type.code == TransformationType.Synonymization:
-                            synonymized, term2 = apply_transformation(term2, tr)
+                            synonymized, term2, _ = apply_transformation(term2, tr)
```

Comment:
@cmungall  Question: At this step the returned  `qualifier` value is ignored. If not, would it influence `rel`
```
rel = RelationshipToTerm(
                        predicate=pred,
                        element=curie,
                        element_term=term,
                        pipeline=pipeline.name,
                        synonymized=synonymized,
                    )
```

Reply:
This is why I didn't want a special purpose `synonymized`, we end up with a lot of ad-hoc additional attributes. Pipelines and transformations were always adequate for this task.